feature/eks-cluster-creation/vars.tf

```diff
@@ -40,10 +65,10 @@ variable "worker1_data" {
   })
   default = {
     name = "eks-worker-1"
-    instance_type = "t2.small"
-    asg_desired_capacity = 2
+    instance_type = "t2.medium"
+    asg_desired_capacity = 1
     asg_min_size = 1
-    asg_max_size = 5
+    asg_max_size = 10
```

Comment:
Maybe should we only declare variables in vars.tf file and define the variables in terraform.tfvars? It could be nice to have  one standard for AKS and EKS.

Reply:
Good point, i just moved them into similarly created dir in ~/docs/design-docs/